<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-5OD0AA6" failures="5" tests="20" name="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="324.471" errors="0" timestamp="2024-11-07T22:03:39 CET" skipped="0">
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="20.900"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="12.385"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="13.574"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="23.299"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="6.887"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="17.279"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="7.390"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="7.888"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="22.136">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.WalidacjaWysyłaniaWiadomości_contactPage_TDD(TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- WalidacjaWysyłaniaWiadomości_contactPage_TDD -->
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="22.831"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="22.748"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="8.419"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="13.894"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="9.949"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="21.269">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.WalidacjaWysyłaniaWiadomości_contactPage_TDD(TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- WalidacjaWysyłaniaWiadomości_contactPage_TDD -->
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="20.884">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.WalidacjaWysyłaniaWiadomości_contactPage_TDD(TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- WalidacjaWysyłaniaWiadomości_contactPage_TDD -->
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="13.804"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="15.217"/>
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="22.465">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.WalidacjaWysyłaniaWiadomości_contactPage_TDD(TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- WalidacjaWysyłaniaWiadomości_contactPage_TDD -->
  <system-out/>
  <testcase classname="testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD" name="WalidacjaWysyłaniaWiadomości_contactPage_TDD" time="21.253">
    <failure message="null" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.WalidacjaWysyłaniaWiadomości_contactPage_TDD(TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- WalidacjaWysyłaniaWiadomości_contactPage_TDD -->
  <system-out/>
</testsuite> <!-- testCases.TC_131_WalidacjaWysyłaniaWiadomości_contactPage_TDD -->
